docs/transaction-path.md

```diff
@@ -2,36 +2,10 @@
 
 Represents a single transaction path.
 
-## Methods
+## Properties
```

Comment:
Why you consider them properties?

Reply:
I was thinking that we might not want methods anymore on this object, and have everything pre-filled as properties instead. Do you think we should e.g. make `apps` a method to instantiate the `App` objects on demand?

Reply:
I agree, let's keep it simple. 

I just thought, does make sense to have a method to get a gas estimation of the transaction path?